components/packaging/rpm/Makefile

```diff
@@ -1,9 +1,13 @@
 ARCH=$(shell uname -m)
 ALPINE_IMG:=$(shell $(CURDIR)/../detect_alpine_image)
+KERNEL:=$(shell docker version --format '{{.Server.Os}}')
```

Comment:
just noticed this; this doesn't contain _kernel_, but _operating system_ - given that this is to make an RPM (or `.deb` in the other case), this could just be hardcoded as `linux`

```
$ docker version --format '{{.Server.Os}}'
linux
```